src/adminBase.js

```diff
@@ -120,15 +130,33 @@ export default class AdminBase extends HasModels {
    * @return {array} 
    */
   modelPopulate () {
-    console.log('returning', this.opts.modelPopulate)
     return this.opts.modelPopulate || []
   }
 
+  /**
+   * Returns a hash of currently used models.
+   * @return {[type]} [description]
+   */
   modelNames () {
     let ret = {}
     ret[this.model()] = this.model()
     return ret;
   }
+
+  /**
+   * Allow upload of models by file type
+   * @return {string}
+   */
+  uploadType() {
+    return this.opts.uploadType || null
+  }
+  /**
+   * Allow upload of models by file type
+   * @return {string}
+   */
+  uploadOptions() {
+    return this.opts.uploadOptions || {}
+  }
```

Comment:
Same as above - just out of curiosity, what are the use cases for uploads outside of data-loader?